templates/webpack.vue.js

```diff
@@ -150,7 +150,7 @@ module.exports = env => {
         },
         module: {
             rules: [{
-                    test: entryPath,
+                    test: new RegExp(entryPath + ".(js|ts)"),
```

Comment:
Maybe you should escape some characters from the path as they might have different meaning in RegExp - `[ \ ^ $ . | ? * + ( )`.

Reply:
Thanks, @KristianDD!

I reverted the commit which resolved the full path for entry module and causes this issue. Also, I updated the regex which uses a relative path and doesn't need to escape characters.